client/certs/money-tree.tech/cert.pem

```diff
@@ -1,56 +0,0 @@
-QlMLmeCpUL6WdP6UkF3mDFNrqjqhNZdUpZoBa9Oh9cx5/si4KJ96toj/A2246o61
-yR+Dunj6tL49znz2KQatd8B/HgtU6C7iBQ+NgQdIE01tKSnebje/bsHzeZMhD0Tl
-fWGe7x11sCvzlbSqthaS2cMCAwEAAaOCAkswggJHMA4GA1UdDwEB/wQEAwIFoDAd
-BgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwDAYDVR0TAQH/BAIwADAdBgNV
-HQ4EFgQUP+ARPBwL7fgDxQu2ol+fgJTk6cwwHwYDVR0jBBgwFoAUFC6zF7dYVsuu
-UAlA5h+vnYsUwsYwVQYIKwYBBQUHAQEESTBHMCEGCCsGAQUFBzABhhVodHRwOi8v
-cjMuby5sZW5jci5vcmcwIgYIKwYBBQUHMAKGFmh0dHA6Ly9yMy5pLmxlbmNyLm9y
-Zy8wGgYDVR0RBBMwEYIPbW9uZXktdHJlZS50ZWNoMEwGA1UdIARFMEMwCAYGZ4EM
-AQIBMDcGCysGAQQBgt8TAQEBMCgwJgYIKwYBBQUHAgEWGmh0dHA6Ly9jcHMubGV0
-c2VuY3J5cHQub3JnMIIBBQYKKwYBBAHWeQIEAgSB9gSB8wDxAHYARJRlLrDuzq/E
-QAfYqP4owNrmgr7YyzG1P9MzlrW2gagAAAF3A9oOqQAABAMARzBFAiEAsKMk2190
-ecjn3d6skgBDHAIvgUtysvizD8aW2PVqX6cCIGVWtF5bzcmRBGgXa9YQ2tGDjLu5
-tZOof94jYdSx9vaWAHcA9lyUL9F3MCIUVBgIMJRWjuNNExkzv98MLyALzE7xZOMA
-AAF3A9oOnQAABAMASDBGAiEAwNWZDz1+1fE1Hv58M+HxMsIa36M3uOra7W4p9NaM
-AKQCIQCsjOXkFtcl5QNWYih2ZC+AY1Mu2x/kqQSQPUj0xb51EzANBgkqhkiG9w0B
-AQsFAAOCAQEABgOGOf3ZkRKJvrS9Yqu/PlqRzeQ1ok0JNxx35ql5dzzP7aRv0tqI
-gK9cQML5Mc1Crm7qYQ8ResF2Bh0zGwLzNmUiFjZpxAEvjIXunSiC8k85fNn8Skgs
-Ro6WxDx1hqvOrW8zpETMSu3rrCeVHk0e2Mtx+RX/SWn0MQz8U4QMo/uovi45Y5O+
-sB/4dzOb40qnkep4qFVHhDwdmdn9p9j3UYhjyFo/LtnpYsEU6dRL0gDSrI7kFk4i
-dvCmibJDPkafS88HIiVG7NUWc5z8oiPtvDtHYPRwsTPGtJDqGAcO2tyr2Af9RU/T
-7LG+IgxVp+k7QD/DQsP+uAgb1ltVkluEOA==
------END CERTIFICATE-----
------BEGIN CERTIFICATE-----
-MIIEZTCCA02gAwIBAgIQQAF1BIMUpMghjISpDBbN3zANBgkqhkiG9w0BAQsFADA/
-MSQwIgYDVQQKExtEaWdpdGFsIFNpZ25hdHVyZSBUcnVzdCBDby4xFzAVBgNVBAMT
-DkRTVCBSb290IENBIFgzMB4XDTIwMTAwNzE5MjE0MFoXDTIxMDkyOTE5MjE0MFow
-MjELMAkGA1UEBhMCVVMxFjAUBgNVBAoTDUxldCdzIEVuY3J5cHQxCzAJBgNVBAMT
-AlIzMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuwIVKMz2oJTTDxLs
-jVWSw/iC8ZmmekKIp10mqrUrucVMsa+Oa/l1yKPXD0eUFFU1V4yeqKI5GfWCPEKp
-Tm71O8Mu243AsFzzWTjn7c9p8FoLG77AlCQlh/o3cbMT5xys4Zvv2+Q7RVJFlqnB
-U840yFLuta7tj95gcOKlVKu2bQ6XpUA0ayvTvGbrZjR8+muLj1cpmfgwF126cm/7
-gcWt0oZYPRfH5wm78Sv3htzB2nFd1EbjzK0lwYi8YGd1ZrPxGPeiXOZT/zqItkel
-/xMY6pgJdz+dU/nPAeX1pnAXFK9jpP+Zs5Od3FOnBv5IhR2haa4ldbsTzFID9e1R
-oYvbFQIDAQABo4IBaDCCAWQwEgYDVR0TAQH/BAgwBgEB/wIBADAOBgNVHQ8BAf8E
-BAMCAYYwSwYIKwYBBQUHAQEEPzA9MDsGCCsGAQUFBzAChi9odHRwOi8vYXBwcy5p
-ZGVudHJ1c3QuY29tL3Jvb3RzL2RzdHJvb3RjYXgzLnA3YzAfBgNVHSMEGDAWgBTE
-p7Gkeyxx+tvhS5B1/8QVYIWJEDBUBgNVHSAETTBLMAgGBmeBDAECATA/BgsrBgEE
-AYLfEwEBATAwMC4GCCsGAQUFBwIBFiJodHRwOi8vY3BzLnJvb3QteDEubGV0c2Vu
-Y3J5cHQub3JnMDwGA1UdHwQ1MDMwMaAvoC2GK2h0dHA6Ly9jcmwuaWRlbnRydXN0
-LmNvbS9EU1RST09UQ0FYM0NSTC5jcmwwHQYDVR0OBBYEFBQusxe3WFbLrlAJQOYf
-r52LFMLGMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjANBgkqhkiG9w0B
-AQsFAAOCAQEA2UzgyfWEiDcx27sT4rP8i2tiEmxYt0l+PAK3qB8oYevO4C5z70kH
-ejWEHx2taPDY/laBL21/WKZuNTYQHHPD5b1tXgHXbnL7KqC401dk5VvCadTQsvd8
-S8MXjohyc9z9/G2948kLjmE6Flh9dDYrVYA9x2O+hEPGOaEOa1eePynBgPayvUfL
-qjBstzLhWVQLGAkXXmNs+5ZnPBxzDJOLxhF2JIbeQAcH5H0tZrUlo5ZYyOqA7s9p
-O5b85o3AM/OJ+CktFBQtfvBhcJVd9wvlwPsk+uyOy2HI7mNxKKgsBTt375teA2Tw
-UdHkhVNcsAKX1H7GNNLOEADksd86wuoXvg==
------END CERTIFICATE-----
```

Comment:
you could have kept the certificate here, this will be public anyway.

Reply:
I think I'll remove them anyway just for security reasons. Let's keep everything in the VM.